lib/queue_classic_plus/base.rb

```diff
@@ -165,5 +189,26 @@ def self.deserialized(args)
     def self.execute(sql, *args)
       QC.default_conn_adapter.execute(sql, *args)
     end
+
+    def self.define_callback(wrapped_method, type, callback_method, &_block)
```

Comment:
Is there a better name than `wrapped_method`?

Reply:
Renamed it to `name` because it's the name of the callback and isn't necessarily tied to a method.
Copied from https://github.com/rails/rails/blob/099f979dc72a0c284d27e581a4e8ec5c9fe1cfb2/activesupport/lib/active_support/callbacks.rb#L776